raiden/blockchain_events_handler.py

```diff
@@ -150,7 +152,7 @@ def create_channel_new_state_change(
             reveal_timeout=reveal_timeout,
             payment_channel_proxy=channel_proxy,
             opened_block_number=block_number,
-            fee_schedule=replace(raiden.config["default_fee_schedule"]),
+            fee_schedule=fee_schedule,
```

Comment:
I think this is wrong, we want a copy of the default, not a reference to the default

Reply:
`fee_schedule` is a copy of the default, see the call to this function in `handle_channel_new` below.